src/meshcat/geometry.py

```diff
@@ -388,3 +427,16 @@ def PointCloud(position, color, **kwargs):
         PointsGeometry(position, color),
         PointsMaterial(**kwargs)
     )
+
+
+def SceneText(text, **kwargs):
+    if 'width' in kwargs and 'height' in kwargs:
```

Comment:
I don't think this is quite right. If the user only provides `width` or `height` then that argument will be ignored. Can't we just do: 

```python
def SceneText(text, width=10, height=10, **kwargs): 
  ...
```
?

Reply:
Yep, agreed. Changed it.